code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -68,6 +69,12 @@
 		B.basecolor = color
 		B.update_icon()
 
+/datum/reagent/slimejelly/reaction_mob(mob/living/M, method, volume)
+	if(method == REAGENT_INGEST && iscarbon(M) && M.mind?.has_antag_datum(/datum/antagonist/vampire))
+		M.reagents.remove_reagent("slimejelly", 1)
+		volume++ // Bit higher than base, wish it could be higher, but this stops slimes from gaming to get more blood out of their blood
+		M.set_nutrition(min(NUTRITION_LEVEL_WELL_FED, M.nutrition + 10))
+		M.blood_volume = min(M.blood_volume + round(volume, 0.1), BLOOD_VOLUME_NORMAL)
```

Comment:
doesnt this do the opposite of what the comment suggests? if you drank 10u of slime jelly as a slime vamp, you'd get 11u of blood? 

ALSO, this code will break normal slime people from regaining blood because it doesnt call the parent.

Reply:
It calls parent now, fixing your second issue. And yeah, you're right about the comment being wrong, it does result in slimes getting slightly more blood out of their blood, I'll update the comment for now